testing/minikube/binderhub_config.py

```diff
@@ -8,3 +8,4 @@
 c.BinderHub.hub_url = 'http://{}:30123'.format(minikube_ip)
 c.BinderHub.hub_api_token = 'aec7d32df938c0f55e54f09244a350cb29ea612907ed4f07be13d9553d18a8e4'
 c.BinderHub.use_registry = False
+c.BinderHub.build_namespace = 'binder-test'
```

Comment:
Just double checking: On mybinder.org this gets set via an environment variable which is set via the helm chart or do we need to adjust something in mybinder.org-deploy before this works?

Reply:
This was just missing from the testing environment. When you run minikube, it was doing builds in 'default' because this was unspecified. The chart is already doing everything right.